python_modules/dagster-ext/dagster_ext/__init__.py

```diff
@@ -254,8 +257,8 @@ def _env_var_to_param_name(env_var: str) -> str:
     return env_var[len(_ENV_KEY_PREFIX) :].lower()
 
 
-def is_dagster_orchestration_active() -> bool:
-    return _param_from_env_var("is_orchestration_active")
+def is_launched_ext_process() -> bool:
```

Comment:
Since data scientists etc who don't know anything about dagster or ext might see this in their scripts, I think it should have "dagster" in the name